docs/sdk_reference.rst

```diff
@@ -0,0 +1,13 @@
+=================================================
+Python SDK Reference
+=================================================
+
+.. toctree::
+    :maxdepth: 4
+
+    oled
+    keyboard
+    case_buttons
+    pulse
+    pma
+    parameters
```

Comment:
Not sure I love this order. Perhaps a bette way to think of grouping things might be:

* Using the pi-top itself (e.g. buttons, OLED, reading battery, etc.)
* Doing physical computing with pi-top [4] (PMA, parameters) (also, shouldn't parameters be under pma anyway?)
* Using peripherals (e.g. pi-topPULSE/pi-topPROTO+)
* Making use of additional helper classes (e.g. keyboard, speech?)

Reply:
Not sure about this... might be confusing for someone coming to find something specific about the OLED (for example).

I do like the sections you proposed though, I think new users would appreciate them... lets discuss on standup

Reply:
Right now I think we have it describing each section of the source code, which means it's tightly coupled to the code. I think it would be better to think about how people are going to want to use it and restructure around that.
Let's ask ourselves 'what questions are people going to ask when they arrive here? what are they trying to do, and what are they looking for to find answers?'

Reply:
Marking as resolved - will move this to another Issue